client/state/simple-payments/product-list/actions.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import {
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_RECEIVE,
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_RECEIVE_UPDATE,
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_RECEIVE_DELETE,
+} from 'state/action-types';
+
+export function receiveProductsList( siteId, numOfProducts, posts ) {
```

Comment:
Maybe a better name for this is `fetchProductsList`, `listProductsList`, I'm sure you'll come up with something better, but something along those lines.